src/devices/SoftwareSpi/SoftwareSpi.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Device.Gpio;
+using System.Device.Spi;
+
+namespace Iot.Device.Spi
```

Comment:
Should the namespace be `Iot.Device.Spi.Devices`?  I guess it fits and not redundant like we kept main `SpiDevice`.

Reply:
I'm expecting this type will eventually get moved to SpiDevice.Create and became internal. I'd just leave this as is

Reply:
Oh that's right.  I recall similar chat for PWM.